src/observable/BoundCallbackObservable.ts

```diff
@@ -138,11 +138,19 @@ function dispatch<T>(state: { source: BoundCallbackObservable<T>, subscriber: Su
   self.add(subject.subscribe(subscriber));
 }
 
-function dispatchNext({ value, subject }) {
+interface DispatchNextArg<T> {
+  subject: AsyncSubject<T>;
+  value: T;
+}
+function dispatchNext<T>({ value, subject }: DispatchNextArg<T>) {
```

Comment:
Umm, it would be just a matter of taste. But I don't have any strong options in my changed area. I'll change to the style to destructure inside of codes.